genderdictionary.py

```diff
@@ -0,0 +1,208 @@
+
+import csv
+with open('C:/Users/tickc/OneDrive/Documents/GitHub/gender/data/country_pop.csv') as g:
```

Comment:
Same comment on hardcoded filepaths as above. If these files are no longer used/needed, then better remove them from the repo than leave them in an unusable state for others.